### DISCUSSION TITLE
HID profile to control IOs

### DISCUSSION THREAD

**Discussion by maszkows** [NicoHood/HID#360] [Ideas]
Hello,

i have a questions:
is it possible with your library to set HID profile to control from HOST (PC) -> ATmega32u4 state if IOs???
for example to contro connectedl LED or attached relay etc.

br
Dan

**Comment by NicoHood** [comment]
You can control the Keyboard LEDs. If you want to send more data, use the Arduino Serial Interface. RawHID is currently not working, I think
